src/control/go.mod

```diff
@@ -5,7 +5,6 @@ module github.com/daos-stack/daos/src/control
 // - debian packaging version checks: debian/control
 // Scons uses this file to extract the minimum version.
 go 1.21
-toolchain go1.23.0
```

Comment:
Just curious: Why was this change made?

Reply:
that change needs to be removed, done

Reply:
Oh, okay. I don't really know why it was added nor why it was removed...

Reply:
I see. It was not easy to find this out from the Git commit messages. Thanks for the info.